dev/run_dev.R

```diff
@@ -9,4 +9,4 @@ golem::detach_all_attached()
 golem::document_and_reload()
 
 # Run the application
-run_app()
+run_app() %>% print()
```

Comment:
I find that without passing `run_app` to `print`, the app does not run when I source `run_dev.R`.